### PR TITLE
Update DESCRIPTION

### PR DESCRIPTION
**Pull request by elray1**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+           family = "Ray",
            role = c("aut", "cre"),
-           email = "first.last@example.com",
-           comment = c(ORCID = "YOUR-ORCID-ID"))
+           email = "elray@umass.edu")
 Description: Versioned Data for COVID-19
 License: MIT
 Imports:
-  dplyr, tibble, tidyr, MMWRweek, lubridate, purrr, magrittr, xml2, rvest, RSocrata, here, httr
+  dplyr, here, httr, lubridate, magrittr, MMWRweek, purrr, readr, RSocrata, rvest, tibble, tidyr, xml2
```
Comment (bsweger) [inline DESCRIPTION:12]: alphabetized dependencies! ✨

File: .github/workflows/pr_unittests.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ jobs:
           key: r-${{ hashFiles('DESCRIPTION') }}
       - name: Install dependencies
         run: |
-          install.packages(c("remotes","devtools","readr","covidcast"))
+          install.packages(c("remotes","devtools","covidcast"))
```
Comment (bsweger) [inline .github/workflows/pr_unittests.yaml:24]: Is `covidcast` also something that should be listed in the "imports" section of the DESCRIPTION?

**Review by bsweger — COMMENTED** [review]
Looks good--just one small question about `covidCast`

File: .github/workflows/pr_unittests.yaml (hunk 2 above)
Comment (elray1) [reply .github/workflows/pr_unittests.yaml:24]: Some thoughts:
- People could happily use this package without using any functionality that requires covidcast, so i'd rather not require installation of that package...?
- however, we should probably check for whether that package is installed before trying to use it, which I bet we don't right now.  i'd move to file a separate issue for that.

Seem reasonable?

**Pull request merged**